scvi/model/base/_base_model.py

```diff
@@ -834,6 +820,16 @@ def view_anndata_setup(
 class BaseMinifiedModeModelClass(BaseModelClass):
     """Abstract base class for scvi-tools models that can handle minified data."""
 
+    def __init__(self, adata: Optional[AnnOrMuData] = None):
+        data_is_minified = (
+            adata is not None and _get_adata_minify_type(adata) is not None
+        )
+        if data_is_minified and not issubclass(type(self), BaseMinifiedModeModelClass):
+            raise NotImplementedError(
+                f"The {type(self).__name__} model currently does not support minified data."
+            )
+        super().__init__(adata)
+
```

Comment:
If I recall, we needed this in the main __init__ in case a user passes a minified anndata to a non-supported model.

Reply:
Ah I see. I'll revert those changes then and close the issue